.gitignore

```diff
@@ -19,3 +19,5 @@ profile.json
 .rust-version
 coverage/
 docs/
+**/**/charts/**.tgz
+values-publisher-env.yaml
```

Comment:
I'm automatically generating the secrets before running Tilt to use locally in our cluster setup by getting them from the `.env` file; because of that we need to ignore this file here